openvdb/tools/PointPartitioner.h

```diff
@@ -918,11 +878,49 @@ inline void partition(
     PointIndexType* index = pointIndices.get();
     for (size_t n = 0; n < numSegments; ++n) {
         indexArray.push_back(index);
-        index += offestSegments[n]->size();
+        index += offsetSegments[n]->size();
     }
 
+    // compute leaf node origin for each page
+
+    pageCoordinates.reset(new Coord[pageCount]);
+
+    tbb::parallel_for(segmentRange,
+        [&](tbb::blocked_range<size_t>& range)
+        {
+            for (size_t n = range.begin(); n < range.end(); n++)
+            {
+                Index segmentOffset = segmentOffsets[n];
+                PointIndexType* indices = pageIndexArrays[n].get();
+
+                const Coord& segmentCoord = segmentCoords[n];
+
+                tbb::blocked_range<size_t> copyRange(0, pageOffsetArrays[n][0] - 1);
```

Comment:
A comment here that the array size is stored in the first element, otherwise it's really not clear

Reply:
I'll put it into a new temporary variable for now to make it clearer, though I believe this is changed in the next round of refactoring.

Reply:
Fixed.